docs/arch/README.md

```diff
@@ -0,0 +1,66 @@
+## MessageHub Trigger Provider Architecture
+
+### Create Trigger Feed
+![MessageHub Trigger Create](images/Arch-Provider-MHV1-Create.png)
+
+**Scenario:** User wants to create a trigger `trigger1` for MessageHub service instance `instance1`, using Credentials `Credential-1` and use rule `rule1` to invoke action `action1` with messages from topic `topic1`.
+
+1. Developer creates a MessageHub service `instance1`
+2. Developer creates topic `topic1` in MessageHub service `instance1`
+3. Developer creates Credential key `Credential-1` for MessageHub `instance1`
+4. Developer creates trigger `trigger1` on OpenWhisk, the trigger stores the annotation `feed` with the feedAction name from system package or binded package.(`/whisk.system/messagingWeb/messageHubFeed`).
+5. Developer invokes action feedAction to create trigger feed passing input parameters (lifeCycle:`CREATE`, trigger1, Credentials1, Options:`topic1`)
```

Comment:
Who is `developer` here: "**Developer** invokes action feedAction" ? May need to clarify.